software/git.txt

```diff
@@ -172,6 +181,98 @@ Note: Do not squash commits that you’ve already shared with others. You are
       changing history and it will cause trouble for others.
 
 
+[4.0] Example 3: Writing a new KISS Wiki article.
+________________________________________________________________________________
+
+Note: In order to add a new article, you will need a GitHub user account.
```

Comment:
Pages can also be submitted via email for those without a GitHub account (to dylan@k1ss.org). :+1: